src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c

```diff
@@ -263,10 +263,8 @@ void EnPoSisters_MatchPlayerY(EnPoSisters* this, PlayState* play) {
     // equalize to player height
     Math_ApproachF(&this->actor.world.pos.y, player->actor.world.pos.y + 5.0f, 0.5f, 3.0f);
 
-    if (this->floatingBobbingTimer == 0) {
+    if ((u32)this->floatingBobbingTimer == 0) {
```

Comment:
this is still kinda fake tho

Reply:
Did you try `!this->floatingBobbingTimer`?

Reply:
`!this->floatingBobbingTimer` is probably the correct solution, yeah. However I thought we decided it was fine to use `u32` casts to still keep the explicit comparison to 0 (we at least have multiple examples of it in the repo already).